SonarQube.TeamBuild.PostProcessor/CoverageReportProcessor.cs

```diff
@@ -5,15 +5,24 @@
 // </copyright>
 //-----------------------------------------------------------------------
 
-using System;
 using SonarQube.Common;
 using SonarQube.TeamBuild.Integration;
+using System;
+using System.Diagnostics;
 
 namespace SonarQube.TeamBuild.PostProcessor
 {
     public class CoverageReportProcessor : ICoverageReportProcessor
     {
-        public bool ProcessCoverageReports(AnalysisConfig config, TeamBuildSettings settings, ILogger logger)
+        private AnalysisConfig config;
+        private TeamBuildSettings settings;
+        private ILogger logger;
```

Comment:
Do these three parameters need to be stored? This class is a shim that delegates the work to the "real" instance of the coverage report processor. It doesn't seem to use these variables except in the Initialise method.